.github/workflows/phpunit.yml

```diff
@@ -43,6 +44,12 @@ jobs:
           key: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
           restore-keys: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
 
+      - name: "Fake PHP version"
+        if: ${{ matrix.php-version == '8.1' }}
+        run: |
+          composer config platform.php 8.0.99
+          composer update --lock
```

Comment:
This hack can be removed as soon as all of our dependencies declare compatibility with PHP 8.1.

Reply:
Prophecy seems to be released and so this should not longer be needed :)

Reply:
`laminas/laminas-coding-standard` would need to be bumped as well. But that should probably be done in a separate PR.

Reply:
#98

Reply:
#98 merged - I think we need to upgrade some of the CI tooling, but this one can probably be merged as-is after a rebase

Reply:
To remove this hack, #98 was actually enough. I've removed it after the rebase.